snappiershot/serializers/utils.py

```diff
@@ -149,16 +149,16 @@ def get_snapshot_file(test_file: Path, suffix: str) -> Path:
     return snapshot_directory.joinpath(test_file.name).with_suffix(suffix)
 
 
-def is_instanced_object(value: Any) -> bool:
-    """ Check if the input value is an instanced object, i.e. an instantiated class. """
+def is_class_object(value: Any) -> bool:
+    """Check if the input value is an instanced object, i.e. an instantiated class."""
     is_type = inspect.isclass(value)
     is_function = inspect.isroutine(value)
```

Comment:
You're using inspect.isroutine to catch both functions and methods?

Reply:
i didnt write that part of the code, i just changed the name of the method. i cant claim to understand it, but it seems to do a good job while debugging of knowing whats a type object and whats a class object via that inspect.isroutine